sematic/runners/local_runner.py

```diff
@@ -122,14 +129,37 @@ def _seed_graph(self, future: AbstractFuture):
         if self._rerun_from_run_id is None:
             super()._seed_graph(future)
         else:
-            self._seed_from_clone(future, self._rerun_from_run_id, self._rerun_mode)
+            self._seed_from_existing(future, self._rerun_from_run_id, self._rerun_mode)
 
-    def _seed_from_clone(
+    def _reenter_inline_runs(self, futures, runs):
+        """During runner reentry, correct the state of any inline futures/runs.
+
+        A correction may be needed if the future was executing in the runner when the
+        runner restarted. In this case, this method will mark the future and run for
+        retry. The update to the future and run happen in-place.
+        """
+        for future in futures:
+            if not future.props.standalone and future.state == FutureState.SCHEDULED:
+                # The future was executing in the runner when the runner restarted.
+                # So we need to start it again.
+                logger.warning(
+                    "The future %s was executing in the runner when the runner "
+                    "restarted. Restarting execution of the future.",
```

Comment:
Please also add a disclaimer that not all future properties could be copied (link the issue), and that execution behavior might not be what was intended.

Reply:
done